### ISSUE TITLE
Lack of social initiations

### ISSUE THREAD

**Issue by bcoleman96** [obophenotype/human-phenotype-ontology#7318]
**Preferred term label:**
Lack of social initiations

**Synonyms**

**Definition (free text, please give PubMed ID)**
Failure of a person to initiate social interactions with others. For a social interaction to have been initiated by someone, there must have been an absence of social interactions from either party for a period. The interaction must also be unprompted from another party.
PMID: 18064552

**Parent term (use hpo.jax.org/app)**
Impaired social interactions
HP:0000735

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**

**Your nano-attribution (ORCID)**
0000-0002-4422-1708

**Comment by pnrobinson** [comment]
added as HP:4000080
